common/tools/dev-tool/src/commands/samples/checkNodeVersions.ts

```diff
@@ -13,6 +13,8 @@ import { S_IRWXO } from "constants";
 import { resolveProject } from "../../util/resolveProject";
 import { findSamplesRelativeDir } from "../../util/findSamplesDir";
 
+const defaultVersions = [14, 16, 17, 18];
```

Comment:
```suggestion
const defaultVersions = [14, 16, 18];
```

I think we can drop 17 entirely. My understanding is we originally used it as a placeholder until 18 was released. It's not a LTS though, so we don't guarantee anything. :)